en/docs/administer/logging-and-monitoring/logging/setting-up-logging-per-api.md

```diff
@@ -0,0 +1,62 @@
+---
+template: templates/swagger.html
+---
+# Setting up logging per API in API Manager
+
+!!! note
+    Enabling logs per API is only available after the WUM update with the timestamp "1600491803101"
+
+In a production envrionment, having the feasbility to monitor HTTP message flow for a given API is important in identifying issues in the HTTP message flow.
```

Comment:
HTTP message flow -> the HTTP message flow

Reply:
fixed